app/mailers/diy_intake_email_mailer.rb

```diff
@@ -4,14 +4,18 @@ def high_support_message(diy_intake:)
     @diy_intake = diy_intake
     service = MultiTenantService.new(:gyr)
     attachments.inline['logo.png'] = service.email_logo
+
+    verifier = ActiveSupport::MessageVerifier.new(Rails.application.secret_key_base)
+    signed_email = verifier.generate(diy_intake.email_address)
```

Comment:
consider adding shared example for testing that the unsubscribe link works (has a signed email) i.e. `shared_example :has_a_working_unsubscribe_link` for all mailers